app/src/components/TimeLine.js

```diff
@@ -59,10 +73,25 @@ function TimeIndicator({ dayStart }) {
         ? (currTime.getHours() - dayStart.getHours()) * 2 + 2
         : (currTime.getHours() - dayStart.getHours()) * 2 + 1;
 
-    console.debug({ row }); //TEST
+    // console.debug({ row }); //TEST
     setIndicatorRender(`${row}`);
   }
 
+  function getCurrentTime() {
+    let currTime = new Date();
+    let hour =
+      currTime.getHours() <= 12
+        ? currTime.getHours()
+        : currTime.getHours() - 12;
+    let minute =
+      currTime.getMinutes() < 10
+        ? `0${currTime.getMinutes()}`
+        : currTime.getMinutes();
+    let amPM = currTime.getHours() <= 11 ? "am" : "pm";
+
+    return `${hour}:${minute}${amPM}`;
+  }
```

Comment:
There's a library called MomentJS that's got a ton of really useful date-and-time helper functions that can do a lot of the above for you, so you don't have to write your own.  I know the library is being (or has been) sunsetted, but it's still pretty widely used (my team still uses it all the time).  Alternately, [its website says](https://momentjs.com/docs/#/-project-status/) there may be more modern alternatives available.